pkg/cmd/app.go

```diff
@@ -226,17 +228,26 @@ func newOSWatcher(sigs ...os.Signal) chan os.Signal {
 	return sigChan
 }
 
-func action(c *cli.Context) error {
+func action(c *cli.Context) (err error) {
 restart:
 
```

Comment:
I am opposite pattern like func() (err error) because it reduces the clarity of return values. Please declare err as a variable inside the function.

Reply:
I don't have any strong opinions about named return types, and given there is some shadowing reverting to declaring err within the function body.

Reply:
It looks like in order to return a proper error to the main function I would need to use named return type otherwise the defer function will always cause `action` to return nil resulting in exit status code 0.

So, we have a choice to use named return type AND return non-zero error code or not use it so resulting in exit status `0` most of the time. My preference is to use return type, @glowkey and @nvvfedorov let me know your preference.

Reply:
My preference is for return type AND non-zero error code.